src/containers/myTrees/index.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useEffect } from 'react';
+import MapPage from '../../components/mapPageComponents/mapPage/index';
+import TreeSidebar from '../../components/treeSidebar/index';
+import { Helmet } from 'react-helmet';
+import useWindowDimensions, {
+  WindowTypes,
+} from '../../components/windowDimensions';
+import { C4CState } from '../../store';
+import MobileMapPage from '../../components/mapPageComponents/mobileMapPage';
+import { useDispatch, useSelector, connect } from 'react-redux';
+import { getMapGeoData } from '../../components/mapPageComponents/ducks/thunks';
+import { getAdoptedSites } from '../treePage/ducks/thunks';
+import { MY_TREES_BODY, MY_TREES_TITLE } from '../../assets/content';
+import SlideDown from '../../components/slideDown';
+import {
+  MapViews,
+  SiteFeaturePropertiesResponse,
+  MapGeoDataReducerState,
+} from '../../components/mapPageComponents/ducks/types';
+import { getMySites } from './ducks/selectors';
+
+interface MyTreesStateProps {
+  readonly blocks: MapGeoDataReducerState['blockGeoData'];
+  readonly neighborhoods: MapGeoDataReducerState['neighborhoodGeoData'];
+  readonly sites: MapGeoDataReducerState['siteGeoData'];
+}
+
+const MyTrees: React.FC<MyTreesStateProps> = ({
+  blocks,
+  neighborhoods,
+  sites,
+}) => {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getMapGeoData());
+    dispatch(getAdoptedSites());
```

Comment:
📌 ❓ Do we dispatch to get the map data every time this page/landing/reservations renders? If so, I think we should consider only dispatching if the state isn't complete/hasn't been updated in a while. Not too sure how this works and not in the scope of this ticket but I've noticed it seems to get slower on subsequent reloads

Reply:
Yeah Ryan and I had a conversation about this yesterday. We're thinking of putting in a timer so that the calls are more staggered